spec/features/list_repos_spec.rb

```diff
@@ -1,14 +1,25 @@
 feature "Viewing a list of public repos" do
+  around(:each) { |example| VCR.use_cassette("repos", record: :new_episodes, &example) }
```

Comment:
Not a blocker, but I have had massive issues with VCR in the past. While it's a pain to craft Webmock stubs in the short term, I find it saves a lot of frustration in the long term

Reply:
Wasn't my choice, but I wasn't very eager to rewrite everything in Webmock either 😭